docs/api.rst

```diff
@@ -477,6 +482,11 @@ to handle it, which defaults to print a traceback and ignoring the exception.
     If this occurs increase the :class:`max_messages <Client>` parameter
     or use the :func:`on_raw_message_edit` event instead.
 
+    .. note::
+
+        Not all messages may have content. This is a discord limitation.
```

Comment:
```suggestion
        Not all messages will have the `content` parameter. This is a discord limitation.
```

Reply:
I agree with @abhigyantrips regarding the capitalization of "discord" here, but I feel like changing this to "content parameter" would be inaccurate since the intent applies to other fields as well (namely components, attachments, and embeds).